test/e2e/framework/ssh/ssh.go

```diff
@@ -113,7 +113,7 @@ func makePrivateKeySignerFromFile(key string) (ssh.Signer, error) {
 }
 
 // NodeSSHHosts returns SSH-able host names for all schedulable nodes - this
-// excludes master node. If it can't find any external IPs, it falls back to
+// excludes Control Plane node. If it can't find any external IPs, it falls back to
```

Comment:
I think it's more appropriate to drop references to the master/control-plane entirely, e.g.

> NodeSSHHosts returns SSH-able host names for all schedulable nodes. If it can't find any ....

Reply:
done. thanks for your  comments.
/cc @spiffxp